include/mppic/parameters_handler.hpp

```diff
@@ -65,11 +65,15 @@ class ParametersHandler
   template<typename T>
   static auto as(const rclcpp::Parameter & parameter);
 
+private:
```

Comment:
protected would be preferable in all situations that you don't actively want to prevent people from subclassing this

Reply:
changed to protected